ext/rbs_extension/main.c

```diff
@@ -18,6 +18,16 @@ Init_rbs_extension(void)
   rbs__init_location();
   rbs__init_parser();
 
-  rbs_constant_pool_init(RBS_GLOBAL_CONSTANT_POOL, 0);
+  // Calculated based on the number of unique strings used with the `INTERN` macro in `parser.c`.
+  //
+  // ```bash
+  // grep -o 'INTERN("\([^"]*\)")' ext/rbs_extension/parser.c \
+  //     | sed 's/INTERN("\(.*\)")/\1/' \
+  //     | sort -u \
+  //     | wc -l
+  // ```
+  const size_t num_uniquely_interned_strings = 26;
```

Comment:
In practice, the `rbs_constant_pool_init` implementation will round this up to the next power of 2 (32), so there's some wiggle room to insert a few extra constants without triggering any reallocations.